src/content/1.7/modules/testing/basic-checks.md

```diff
@@ -86,3 +86,34 @@ Using cache file ".php_cs.cache".
 
 Checked all files in 0.085 seconds, 12.000 MB memory used
 ```
+
+## License headers
+
+When the extension is open-source, sharing the license is one important task.
+
+[Header-Stamp](https://github.com/PrestaShopCorp/header-stamp) is a tool that applies a header on all compatible files (PHP, JS...).
+In these headers, details can be found about the author, the license, the original year of publication.
+Everytime there is a change to make, one command would apply it to all your files.
```

Comment:
```suggestion
One simple command will apply the license header to all your files, or update it as necessary.
```

Reply:
I would avoid giving an opinion with `simple`. Readers may have another feeling about this.

What about

```suggestion
One command will apply the license header to all your files, or update it as necessary.
```

Reply:
I agree with @Quetzacoalt91 https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher 👍